runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -1263,7 +1263,7 @@ ROMClassBuilder::computeExtraModifiers(ClassFileOracle *classFileOracle, ROMClas
 		}
 	}
 
-	if ( classFileOracle->getMajorVersion() >= (BCT_Java6MajorVersionShifted >> BCT_MajorClassFileVersionMaskShift) ) {
+	if ( classFileOracle->getMajorVersion() >= (BCT_JavaMajorVersionShifted(6) >> BCT_MajorClassFileVersionMaskShift) ) {
```

Comment:
Please fix the formatting on this modified line (remove extra spaces inside outermost parentheses).

Reply:
Removed.